presto-main/src/test/java/io/prestosql/sql/query/TestColumnMask.java

```diff
@@ -455,14 +455,19 @@ public void testShowStats()
                 USER,
                 new ViewExpression(USER, Optional.of(CATALOG), Optional.of("tiny"), "7"));
 
-        assertThatThrownBy(() -> assertions.query("SHOW STATS FOR (SELECT * FROM orders)"))
-                .hasMessageMatching("\\QSHOW STATS for table with column masking is not supported: orderkey");
-        assertThatThrownBy(() -> assertions.query("SHOW STATS FOR (SELECT orderkey FROM orders)"))
-                .hasMessageMatching("\\QSHOW STATS for table with column masking is not supported: orderkey");
+        assertThat(assertions.query("SHOW STATS FOR (SELECT * FROM orders)"))
+                .containsAll("VALUES " +
+                        "(CAST('orderkey' AS varchar), CAST(NULL AS double), CAST(1 AS double), 0e1, NULL, '7', '7')," +
+                        "(CAST('clerk' AS varchar), 15e3, 1e3, 0e1, NULL, CAST(NULL AS varchar), CAST(NULL AS varchar))," +
+                        "(NULL, NULL, NULL, NULL, 15e3, NULL, NULL)");
+        assertThat(assertions.query("SHOW STATS FOR (SELECT orderkey FROM orders)"))
+                .matches("VALUES " +
+                        "(CAST('orderkey' AS varchar), CAST(NULL AS double), CAST(1 AS double), 0e1, NULL, CAST('7' AS varchar), CAST('7' AS varchar))," +
```

Comment:
`CAST(1 AS double)`

i still think a `statsRow` formatting function could be useful, but it looks good enough